components/widgets/modals/full_screen_modal.tsx

```diff
@@ -12,32 +12,37 @@ import './full_screen_modal.scss';
 // This must be on sync with the animation time in ./full_screen_modal.scss
 const ANIMATION_DURATION = 100;
 
-export default class FullScreenModal extends React.Component {
-    static propTypes = {
+type Props = {
+    show: boolean;
+    onClose: () => void;
+};
+
+export default class FullScreenModal extends React.Component<Props> {
+    public static propTypes = {
         show: PropTypes.bool.isRequired,
         children: PropTypes.node.isRequired,
         onClose: PropTypes.func.isRequired,
     }
 
-    componentDidMount() {
+    public componentDidMount(): void {
```

Comment:
I think you've set eslint-typescript to be too strict. Adding explicit return types to every method, even the void ones is... Tiresome IMHO

Reply:
It is the "recommended" configuration

Reply:
my fault, is not needed the `: void` type

Reply:
Yes, it was a warning in my editor, not an error in the linter